.github/workflows/auto-pr.yml

```
@@ -0,0 +1,27 @@
+name: Auto PR for new versions
+
+on:
+  schedule:
+  - cron: 0 0 * * *
```

```suggestion
  - cron: 0 * * * *
```

Every hour instead of every day? Otherwise I assume people will be posting issues wondering when the next release will be available if it's potentially 23 hours delayed.

Could run it event more often of course, not like it takes a lot of resources/time

---

@nschonni I still think this makes sense, fwiw

---

I think every hour is better 👍

---

or even more often - any issue running this every 10 minutes?